ethcore/src/miner/miner.rs

```diff
@@ -630,6 +629,7 @@ impl Miner {
 		}
 	}
 
+	// TODO: This is only used in authority_round path, and should be refactored to merge with the other seal() path.
```

Comment:
Please open an issue copy to link to it here

Reply:
https://github.com/paritytech/parity-ethereum/issues/10407